projects/packages/my-jetpack/src/products/class-jetpack-ai.php

```diff
@@ -98,6 +103,10 @@ public static function get_current_usage_tier() {
 	 * @return int
 	 */
 	public static function get_next_usage_tier() {
+		if ( ! self::is_site_connected() ) {
```

Comment:
Can you add the same comment from above here too?
// Default to 100 requests if the site is not registered/connected.